### PR TITLE
docs(readme): add license notice and additional info

### PR DESCRIPTION
**Pull request by owlot**
I compared the README.md with our [Template README.md](https://github.com/Arrow-air/svc-template-rust/blob/develop/README.md) and added missing blocks

### CODE REVIEW

**Review by ServiceDog — APPROVED** [review]
@owlot If BUSL == BSL, looks good

File: README.md [hunk 1]
```diff
@@ -78,6 +106,10 @@ Learn more about us:
 - [Arrow Docs](https://www.arrowair.com/docs/intro)
 - [Discord](https://discord.com/invite/arrow)
 
+## LICENSE Notice
+
+Please note that svc-template is under BUSL license until the Change Date, currently the earlier of two years from the release date. Exceptions to the license may be specified by Arrow Governance via Additional Use Grants, which can, for example, allow svc-template to be deployed for certain production uses. Please reach out to Arrow DAO to request a DAO vote for exceptions to the license, or to move up the Change Date.
+
```
Comment (GoodluckH) [inline README.md:112]: Should be svc-cargo?

**Review by GoodluckH — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -78,6 +106,10 @@ Learn more about us:
 - [Arrow Docs](https://www.arrowair.com/docs/intro)
 - [Discord](https://discord.com/invite/arrow)
 
+## LICENSE Notice
+
+Please note that svc-cargo is under BUSL license until the Change Date, currently the earlier of two years from the release date. Exceptions to the license may be specified by Arrow Governance via Additional Use Grants, which can, for example, allow svc-template to be deployed for certain production uses. Please reach out to Arrow DAO to request a DAO vote for exceptions to the license, or to move up the Change Date.
```
Comment (GoodluckH) [inline README.md:111]: There's one more svc-template hiding lol. Other repos may have the same issue.

**Review by GoodluckH — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (owlot) [reply README.md:111]: oh, haha, was too fast, fixing :)

**Review by GoodluckH — APPROVED** [review]

**Pull request merged**